lib/src/clojure_lsp/feature/diagnostics.clj

```diff
@@ -16,6 +16,8 @@
 
 (set! *warn-on-reflection* true)
 
+(def ^:dynamic *uri->filename* nil)
```

Comment:
I'm not sure it's a good idea to start using dynamic vars in the project like that, I think passing as args would be best, otherwise, we would start introducing states between namespaces and that would be dangerous

Reply:
I'll remove this commit, it was an attempt at blind performance optimization before I realized that the mac integration test is failing for everyone!

Reply:
Dropped the commit